mobile/ios/simulator/ios-simulator-application-manager.ts

```diff
@@ -55,8 +55,25 @@ export class IOSSimulatorApplicationManager extends ApplicationManagerBase {
 		return true;
 	}
 
-	public isLiveSyncSupported(appIdentifier: string): IFuture<boolean> {
-		return ((): boolean => {
+	public getApplicationInfo(applicationIdentifier: string): IFuture<Mobile.IApplicationInfo> {
+		return ((): Mobile.IApplicationInfo => {
+			let applicationPath = this.iosSim.getApplicationPath(this.identifier, applicationIdentifier),
```

Comment:
this method shares a lot of code with `isLiveSyncSupported` method below. Can you extract common method that both methods will call?